src/JsonConfig.cpp

```diff
@@ -37,46 +38,30 @@ JsonConfig::JsonConfig(const QString &path) :
 
 bool JsonConfig::load()
 {
-    QFile file(m_path);
-    if (!file.open(QFile::ReadOnly)) {
-        return false;
-    }
-
-    QJsonDocument doc = QJsonDocument::fromJson(file.readAll());
-    QJsonValue schemaVersionsRaw = doc.object().value("schemaVersions");
-    if (!schemaVersionsRaw.isArray()) {
-        qWarning() << "Invalid schemaVersions";
+    QDir dir(m_path);
+    dir.setFilter(QDir::AllEntries | QDir::NoDotAndDotDot);
```

Comment:
It is enough only to look for text files that end with `.sql`.

Reply:
Yeah makes sense, a more narrow filter. Changed.